src/Abp.Dapper/Dapper/Repositories/DapperRepositoryBaseOfTEntityAndTPrimaryKey.cs

```diff
@@ -113,35 +113,35 @@ public override IEnumerable<TEntity> Query(string query, object parameters = nul
             return GetConnection().Query<TEntity>(query, parameters, GetActiveTransaction(), commandTimeout: Timeout);
         }
 
-        public override async Task<IEnumerable<TEntity>> QueryAsync(string query, object parameters = null)
+        public override Task<IEnumerable<TEntity>> QueryAsync(string query, object parameters = null)
         {
-            var connection = await GetConnectionAsync();
-            var activeTransaction = await GetActiveTransactionAsync();
-            return await connection.QueryAsync<TEntity>(query, parameters, activeTransaction, Timeout);
+            var connection = GetConnection();
+            var activeTransaction = GetActiveTransaction();
+            return connection.QueryAsync<TEntity>(query, parameters, activeTransaction, Timeout);
```

Comment:
Maybe the `GetConnectionAsync` and `GetActiveTransactionAsync` async methods should be used.

Reply:
If we do that, QueryAsync can't be intercepted and can't be used. Dapper_Repository_Tests test fails if we use async versions.

Reply:
https://github.com/aspnetboilerplate/aspnetboilerplate/pull/6085/commits/8d7e5978d3d024a25ba418b90844289a1b2055d2